data/harvesting.txt

```diff
@@ -234,3 +234,22 @@ outfit "Uranium"
 	"mass" 1
 	"installable" -1
 	description "Uranium is best known for its use in the nuclear reactors that power large starships. Its most stable isotope U-238 is also used for ultra-strong hull plating and for armor-piercing projectiles."
+
+
+minable "yottrite"
+	sprite "asteroid/yottrite/spin"
+	hull 20000
+	payload "Yottrite" 7
+	explode "smoke" 30
+	explode "bolide" 30
+	explode "final explosion medium"
+
+outfit "Yottrite"
+	plural "Yottrite"
+	category "Special"
+	cost 200000
+	thumbnail "outfit/harvested yottrite"
+	"flotsam sprite" "effect/flotsam yottrite"
+	"mass" 1
```

Comment:
I feel like we could make these seem a little more "special" by not having the same 1 ton mass as every other ore. Perhaps the flotsams blast off of the asteroid in chunks of 2, 3, or even 5 tons.

Reply:
I was going to originally do something with the mass, however I also do want the Kor Sestor to eventually have an asteroid thingy of their own (not necessarily a made-up one like Yottrite, and not really needing to be restricted to them, but more common in their space, since it could add to the planet descriptions mentioning how they mine by yeeting asteroids into planets), and I thought I'd save that for theirs. Could do it for this one too if you really feel like it'd be neato, just felt like it was worth mentioning.

Reply:
It's up to you, as this is just a random idea I had, not something I think needs changed.